eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/http/AbstractLiteClient.java

```diff
@@ -46,4 +53,22 @@ public LiteClientConfig getLiteClientConfig() {
     public void shutdown() throws Exception {
         logger.info("AbstractLiteClient shutdown");
     }
+
+    public CloseableHttpClient setHttpClient() throws Exception {
+        if (!liteClientConfig.isUseTls()) {
+            return HttpClients.createDefault();
+        }
+        SSLContext sslContext = null;
+        try {
+            String protocol = System.getProperty("ssl.client.protocol", "TLSv1.1");
```

Comment:
is there any particular reason we need to default to TLS 1.1 here?

would it be better if we default to TLS 1.2 instead?

Reply:
This is the original code from the SDK. I believe TLS 1.1 is out dated. I can move it TLS 1.2 instead. Thanks for catching it.